### ISSUE TITLE
App won't back out from results page when pressing back button

### ISSUE THREAD

**Issue by NotYeMK** [luk1337/SauceNAO#45]
Example situation: find an image that I want to get a source on. Open the results page and find the result I'm looking for. Come back to the app to the results page. Pressed the android back button. Nothing happens. Press the back button on the top left corner and it goes back to the main page. Press the android back button to go back to the last app I wanted to get the source of the image from. 

Phone: Nothing Phone 2
OS: Nothing OS based on Android 13

**Comment by luk1337** [comment]
Can you record a video with taps enabled?

**Comment by luk1337** [comment]
> Can you record a video with taps enabled?

Reopen once that's available.

**Comment by NotYeMK** [comment]
https://github.com/luk1337/SauceNAO/assets/76539335/6201d3bf-c3e2-4596-8b5f-df0e46fd0dad

Censored it a bit to cover personal stuff and NSFW stuff, but should be sufficient info in the video. Here it is with taps showing. Steps to try and reproduce the issue:

- open image link with saucenao app
- go to page in Pixiv app
- go back to Saucenao by phone's back button
- press the phone's back button to try and go back

At that point there is where it shows up.

**Comment by NotYeMK** [comment]
https://github.com/luk1337/SauceNAO/assets/76539335/7903deca-dcce-4703-9661-b1be58bd27cf

Video to show. Taps enabled with a bit of censoring to cover personal and NSFW stuff. How to get to the issue:

- open image link in Saucenao app
- open source link to source app, which is Pixiv in my case
- use phone's back button to go to back to Saucenao app
- press phone's back button to go back to original app I started in

After that last step is where the issue appears. Going back with the phone's back button doesn't work. I used the button in the top left to go back to show that workaround.
